### PR TITLE
Added __main__.py as the entrypoint for the game

### PR DESCRIPTION
**Pull request by AbooMinister25**

### CODE REVIEW

**Review by Objectivitix — APPROVED** [review]
LGTM! Just fix the lint error so we get a clean check mark lol

File: src/__main__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import pyglet
+
+from game import update
```
Comment (brad90four) [inline src/__main__.py:3]:
```suggestion
from .game import update
```
Testing locally, I needed to add a `.` to run the game with `py -m src`

**Review by brad90four — CHANGES_REQUESTED** [review]
Just this one thing from testing locally

**Review by Oreo4102004 — APPROVED** [review]

**Review by Oreo4102004 — APPROVED** [review]

**Review by Oreo4102004 — APPROVED** [review]

**Review by brad90four — APPROVED** [review]
Thanks for the quick turn-around on it!

**Pull request merged**